src/platform/intel/cavs/lib/pm_runtime.c

```diff
@@ -368,14 +374,24 @@ static inline void cavs_pm_runtime_dis_dsp_pg(uint32_t index)
 		if (tries == 0)
 			tr_err(&power_tr, "cavs_pm_runtime_dis_dsp_pg(): failed to power up core %d",
 			       index);
+		if (flag)
+			pprd->dsp_pwr_sref[index] |= PWRD_BY_HPRO;
```

Comment:
Now bits (flags) are used to indicate active clients, so `sref` is not appropriate name anymore, pls replace with for example `client_bitmap`